### PR TITLE
Operator now supports device files outside the /dev

### PR DESCRIPTION
**Pull request by aravindavk**
- If device file is specified is outside `/dev` then Operator will
  mount the parent directory of file to server container.
- Incremented the Fedora version to 30 so that it will pull the latest
  stable gluster version(6.3)
- `include_uninitialized` argument is removed in latest kubernetes python
  client. So removed from Operator code.

Signed-off-by: Aravinda VK <mail@aravindavk.in>

### CODE REVIEW

File: templates/server.yaml.j2 [hunk 1]
```diff
@@ -97,6 +101,8 @@ spec:
             - name: glusterfsd-datadir
               mountPath: "/bricks"
               mountPropagation: Bidirectional
+            - name: glusterfsd-volfilesdir
+              mountPath: "/var/lib/gluster"
```
Comment (amarts) [inline templates/server.yaml.j2]: What is this? Why is this required?

File: templates/server.yaml.j2 [hunk 2]
```diff
@@ -111,10 +117,16 @@ spec:
...
+{%- if brick_device == "" %}
           hostPath:
             path: "{{ host_brick_path }}"
             type: Directory
-{% else %}
+{%- else %}
           emptyDir: {}
-{% endif %}
+{%- endif %}
+{%- if brick_device_dir != "" %}
```
Comment (amarts) [inline templates/server.yaml.j2:125]: Looks like we need to get the similar logic for 'path' too. Otherwise, path directory is not exported inside glusterfsd container IMO.

File: templates/server.yaml.j2 (hunk 1 above)
Comment (aravindavk) [reply templates/server.yaml.j2]: I saw a log from quotad container, let me recheck. May be a setup issue. If it works without this change I will remove.

File: templates/server.yaml.j2 (hunk 2 above)
Comment (aravindavk) [reply templates/server.yaml.j2:125]: It is already handled for path directory https://github.com/aravindavk/kadalu/blob/master/templates/server.yaml.j2#L114 

Looks like not working as expected, I will debug further.

Comment (aravindavk) [reply templates/server.yaml.j2:125]: Silly mistake... used `or` instead of `and` here https://github.com/aravindavk/kadalu/blob/master/server/glusterfsd.py#L117

Comment (amarts) [reply templates/server.yaml.j2:125]: Cool :+1

**Comment by aravindavk** [comment]
Updated the code and tested with all three cases. Please review

- Provide `device: /dev/vdc` or `device: /dev/loop0`
- Provide `device: /bricks/brick1.img`
- Provide `path: /bricks/brick2`

**Review by amarts — APPROVED** [review]
LGTM
